go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/tedsuo/rata v1.0.0 // indirect
 	github.com/vito/go-interact v1.0.0 // indirect
 	google.golang.org/protobuf v1.26.0
-	k8s.io/apimachinery v0.19.4 // indirect
+	k8s.io/apimachinery v0.19.4
 )
 
-// replace github.com/hashicorp/waypoint-plugin-sdk => ../../waypoint-plugin-sdk
+replace github.com/hashicorp/waypoint-plugin-sdk => github.com/swisscom/waypoint-plugin-sdk v0.0.0-20210426082641-d2f8c888d724
```

Comment:
this should be temporary and we should remove it again once the change on the SDK is merged. But I'm fine with merging it.

Reply:
Actually, I don't mind waiting for a bit until the SDK is updated too. I'll keep this PR open until the SDK PR is merged 🚀